migrations/20210216155935_initial.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable func-names */
+exports.up = function(knex) {
+  // Add extention for handling uuids to postgres
+  return knex.raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";').then(function() {
+    return knex.schema
+      .createTable('universities', (table) => {
+        table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'))
+        table.string('name', 255).notNullable()
+        table.timestamps() //updated_at / created_at
+      })
+  }).then(function() {
+    return knex.schema.createTable('patients', (table) => {
+      table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'))
+      table.string('firstNames', 255).notNullable()
+      table.string('lastName', 255).notNullable()
+      table.string('INE', 11)
+      table.string('otherStudentNumber', 255) // if INE cannot be found
+      table.timestamps() //updated_at / created_at
+
+      // Note : students may not have a university. Ex: BTS students
+      table.uuid('universityId')
+      table.foreign('universityId').references('id').inTable('universities')
+    })
+  }).then(function() {
+    return knex.schema
+        .createTable('psychologists', (table) => {
+          table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'))
+          table.string('adeli').notNullable()
+          table.string('firstNames', 255).notNullable()
+          table.string('lastName', 255).notNullable()
+          table.text('email') // todo will this be a login ?
+          table.string('address')
```

Comment:
je les ai passé en `text` car elles peuvent être longues sur la PR : https://github.com/betagouv/sante-psy/blob/feat/import-data-from-DS-to-PG/migrations/20210216115133_psychologist.js#L8-L24